wcfsetup/install/files/lib/data/comment/CommentEditor.class.php

```diff
@@ -56,4 +56,37 @@ public function updateUnfilteredResponseIDs()
 
         $this->update(['unfilteredResponseIDs' => \serialize($responseIDs)]);
     }
+
+    /**
+     * Updates the counter for responses.
+     * 
+     * @since 6.0
+     */
+    public function updateResponses(): void
+    {
+        $sql = "SELECT      COUNT(*)
+                FROM        wcf" . WCF_N . "_comment_response
+                WHERE       commentID = ?
+                        AND isDisabled = ?";
+        $statement = WCF::getDB()->prepareStatement($sql);
```

Comment:
This should use `WCF::getDB()->prepare()` so that we eventually get rid of `prepareStatement()` unless we really need to.